bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapLink.java

```diff
@@ -91,7 +91,7 @@ public BootstrapLink(final String id, final IModel<T> model, final Buttons.Type
 
         add(icon = newIcon("icon"));
         add(splitter = newSplitter("splitter"));
-        add(label = newLabel("label"));
+        add(label = newLabel("label", model));
```

Comment:
Look on abstractLink :-) There is already a model there for label.

On Mon, May 9, 2016 at 9:23 PM, Gabriel Landon <notifications@github.com>
wrote:

> In
> bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/BootstrapLink.java
> <https://github.com/l0rdn1kk0n/wicket-bootstrap/pull/603#discussion_r62557130>
> :
>
> > @@ -91,7 +91,7 @@ public BootstrapLink(final String id, final IModel<T> model, final Buttons.Type
> >
> >          add(icon = newIcon("icon"));
> >          add(splitter = newSplitter("splitter"));
> > -        add(label = newLabel("label"));
> > +        add(label = newLabel("label", model));
>
> I don't know what a "bodymodel" is.
> Maybe we can simply add a new constructor like :
> public BootstrapLink(final String id, final IModel<T> linkModel, final
> Buttons.Type type, final IModel<?> labelModel)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/l0rdn1kk0n/wicket-bootstrap/pull/603/files/c869292ba1601c1751a451c9da7413f355efe99a#r62557130>
>



-- 
Regards - Ernesto Reinaldo Barreiro